Sources/Sentry/SentrySDK.m

```diff
@@ -42,24 +42,32 @@ + (void)setCurrentHub:(SentryHub *)hub {
 }
  
 + (id)initWithOptions:(NSDictionary<NSString *,id> *)optionsDict {
```

Comment:
```suggestion
+ (id)startWithOptions:(NSDictionary<NSString *,id> *)optionsDict {
```

should this be renamed too?

Reply:
if it's the deprecated method, I'd add a "javadoc"

Reply:
These are the old `init`, if we remove them we have a breaking change.

Reply:
@HazAT yep but whats about a deprecated javadoc? like you did with other methods eg `initWithOptionsObject`